### PR TITLE
Beta/remove beta bc code for hard break

### PR DESCRIPTION
**Pull request by tsmith023**

### CODE REVIEW

File: integration/test_collection_filter.py [hunk 1]
```diff
@@ -695,31 +633,13 @@ def test_time_update_and_creation_time(collection_factory: CollectionFactory) ->
...
 def test_ref_count_filter(collection_factory: CollectionFactory) -> None:
     collection = collection_factory()
     collection.config.add_reference(
         ReferenceProperty(name="ref", target_collection=collection.name)
     )
 
-    if not collection._connection._weaviate_version.is_at_least(
-        1, 23, patch=3
-    ):  # todo change to patch=4 when it lands
+    if not collection._connection._weaviate_version.is_at_least(1, 23, patch=3):
```
Comment (dirkkul) [inline integration/test_collection_filter.py:642]: we can remove this completely? Same for the others

**Review by dirkkul — APPROVED** [review]

File: integration/test_collection_filter.py (hunk 1 above)
Comment (tsmith023) [reply integration/test_collection_filter.py:642]: Ah yes the tests too, missed those!

File: integration/test_collection_filter.py [hunk 2]
```diff
@@ -313,7 +305,7 @@ def test_filters_contains(
             Property(name="uuid", data_type=DataType.UUID),
         ],
     )
-    if not collection._connection._weaviate_version.is_at_least(1, 23, 0) and skip:
+    if skip:
         pytest.skip("not supported in this version")
```
Comment (dirkkul) [inline integration/test_collection_filter.py:309]: this can also go away?

Comment (tsmith023) [reply integration/test_collection_filter.py:309]: Yeh wasn't sure about this one so left, will remove completely now

Comment (dirkkul) [reply integration/test_collection_filter.py:309]: can't remember why it is there, but we moved past "not supported in this version" :)

**Pull request merged**